app/containers/ActionSheet/Item.js

```diff
@@ -6,25 +6,40 @@ import { themes } from '../../constants/colors';
 import { CustomIcon } from '../../lib/Icons';
 import styles from './styles';
 import { Button } from './Button';
+import Avatar from '../Avatar';
 
-export const Item = React.memo(({ item, hide, theme }) => {
+export const Item = React.memo(({
+	item, hide, theme, user, baseUrl, reactionsMode
+}) => {
 	const onPress = () => {
 		hide();
-		item?.onPress();
+		item?.onPress?.();
 	};
 
 	return (
 		<Button
-			onPress={onPress}
+			onPress={reactionsMode ? () => {} : onPress}
 			style={[styles.item, { backgroundColor: themes[theme].focusedBackground }]}
 			theme={theme}
 		>
-			<CustomIcon name={item.icon} size={20} color={item.danger ? themes[theme].dangerColor : themes[theme].bodyText} />
+			{reactionsMode
```

Comment:
IMO the Item should know what to render without having to explicitly say "reaction mode".
For example:
```
{icon ? icon : null}
{avatar ? avatar : null}
```